Source/WebCore/platform/graphics/gstreamer/mse/AppendPipeline.cpp

```diff
@@ -176,7 +177,7 @@ AppendPipeline::AppendPipeline(SourceBufferPrivateGStreamer& sourceBufferPrivate
         GRefPtr<GstPad> identitySrcPad = adoptGRef(gst_element_get_static_pad(m_demux.get(), "src"));
         gst_pad_add_probe(identitySrcPad.get(), GST_PAD_PROBE_TYPE_BUFFER, reinterpret_cast<GstPadProbeCallback>(
             +[](GstPad *pad, GstPadProbeInfo*, AppendPipeline* appendPipeline) {
-                GRefPtr<GstCaps> caps = gst_pad_get_current_caps(pad);
+                auto caps = adoptGRef(gst_pad_get_current_caps(pad));
```

Comment:
My point was rather that there is no point on changing that.

Reply:
No point in fixing a leak?

Reply:
Ah.

AH.

AAAH.

Okay, nvm. I hadn't noticed that. That should be in a different PR if for no other reason that otherwise it may get lost when doing backports.

Reply:
https://github.com/WebKit/WebKit/pull/7495